core/workspace_svg.js

```diff
@@ -431,6 +421,59 @@ Blockly.WorkspaceSvg.prototype.inverseScreenCTM_ = null;
  */
 Blockly.WorkspaceSvg.prototype.inverseScreenCTMDirty_ = true;
 
+/**
+ * Get the marker manager for this workspace.
+ * @return {Blockly.MarkerManager} The marker manager.
+ */
+Blockly.WorkspaceSvg.prototype.getMarkerManager = function() {
+  return this.markerManager_;
+};
+
+/**
+ * Add the cursor svg to this workspaces svg group.
+ * @param {SVGElement} cursorSvg The svg root of the cursor to be added to the
+ *     workspace svg group.
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.setCursorSvg = function(cursorSvg) {
```

Comment:
Is there still a need for these? Can we not instead use ``getMarkerManager().setCursorSvg`` internally? Since this is @package.

Reply:
setParent_ in MarkerSvg requires that the newParent have a setMarkerSvg and setCursorSvg method. Since the parent can be a workspace I think we need it for right now.